test/parallel/test-child-process-max-buffer.js

```diff
@@ -9,7 +10,7 @@ if (process.argv[2] === 'child') {
 } else {
   const cmd = `${process.execPath} ${__filename} child`;
 
-  cp.exec(cmd, {maxBuffer: 10}, common.mustCall((err, stdout, stderr) => {
+  cp.exec(cmd, { maxBuffer: 10 }, common.mustCall((err, stdout, stderr) => {
     assert.strictEqual(err.message, 'stdout maxBuffer exceeded');
   }));
```

Comment:
If we're going to remove the `stderr` test, can we test it here too? Maybe add a third command line argument that indicates stdout or stderr and then call `cp.exec()` inside this callback?